src/kvstore/test/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ add_executable(
     part_test
     PartTest.cpp
     $<TARGET_OBJECTS:kvstore_obj>
+    $<TARGET_OBJECTS:hbasestore_obj>
```

Comment:
UTs under kvstore should NOT depends on hbaseXXX

Reply:
Should Open a new issue to implement HBaseStore in plugin form.